src/main/webapp/WEB-INF/jsp/viewAllSubjectSDVtmp.jsp

```diff
@@ -474,6 +474,8 @@
         function getItems() {
             var sinceLastVerified = $('#sdv-show-type input:checked').val();
             $.get(url + '?changedAfterSdvOnlyFilter=' + sinceLastVerified, function (data) {
+                if (!data.lastVerifiedDate)
```

Comment:
Were we always setting lastVerifiedDate before this feature was implemented?

Reply:
OK changed in the latest commit